src/site/stages/build/process-cms-exports/transformers/node-press_release.js

```diff
@@ -0,0 +1,74 @@
+const moment = require('moment');
+
+const {
+  createMetaTagArray,
+  getDrupalValue,
+  getWysiwygString,
+  isPublished,
+} = require('./helpers');
+
+const transform = entity => ({
+  entityType: 'node',
+  entityBundle: 'press_release',
+  title: getDrupalValue(entity.title),
+  entityMetatags: createMetaTagArray(entity.metatag.value),
+  entityUrl: {
+    path: entity.path[0].alias,
+  },
+  fieldAddress: {
+    locality: entity.fieldAddress[0].locality,
+    // The things in fieldAddress don't get camelCased
+    administrativeArea: entity.fieldAddress[0].administrative_area,
+  },
```

Comment:
It may be worth it to just copy over the `fieldAddress[0]` here and in the schema use the common transformed address that is in Chris's PR

Reply:
Just be sure to `camelCase` all the properties when you do. :+1: